src/quacc/recipes/onetep/_base.py

```diff
@@ -0,0 +1,77 @@
+"""Base jobs for Onetep."""
```

Comment:
1. Please also add a section for ONETEP in the installation [guide](https://github.com/Quantum-Accelerators/quacc/blob/main/docs/install/codes.md).
2. Please add your new recipe to the [recipes list](https://github.com/Quantum-Accelerators/quacc/blob/main/docs/user/recipes/recipes_list.md).